_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw_par.cpp

```diff
@@ -925,10 +925,32 @@ mfxU16 GetMaxNumRef(MfxVideoParam &par, bool bForward)
 
 #endif // (MFX_VERSION >= 1025)
 
+#if defined (MFX_ENABLE_HEVCE_ROI) || defined (MFX_ENABLE_HEVCE_DIRTY_RECT)
+mfxStatus CheckAndFixRect(RectData *rect, MfxVideoParam const & par, ENCODE_CAPS_HEVC const & caps) {
+    mfxU32 changed = 0;
+
+    mfxU32 blockSize = 1 << (caps.BlockSize + 3);
+    changed += CheckRange(rect->Left, mfxU32(0), mfxU32(par.mfx.FrameInfo.Width));
```

Comment:
```0u``` is unsigned int literal.
and please add alignment
```
     changed += CheckRange(rect->Left,  0u, mfxU32(par.mfx.FrameInfo.Width));
     changed += CheckRange(rect->Right, 0u, mfxU32(par.mfx.FrameInfo.Width)); 
```

Reply:
Done.